### PR TITLE
Hmp create

### PR DESCRIPTION
**Pull request by hmparsons-sde**
## Description
Souvenirs Create

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/javascripts/events/domEvents.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import printShows from '../components/printShows';
+import { buildSouvenirs } from '../components/buildSouvenirs';
+import { createSouvenirs, getSouvenirs } from '../helpers/data/souvenirData';
+import showFood from '../components/showFood';
+import getFood from '../helpers/data/foodData';
```
Comment (jrobinson0529) [inline src/javascripts/events/domEvents.js]: getFood needs brackets around it

**Review by jrobinson0529 — DISMISSED** [review]
Wasn't merged properly. Missing create on the other modules.

**Review by chiestroud — APPROVED** [review]
:)

**Review by katherinevfry — APPROVED** [review]
💯

**Review by jrobinson0529 — APPROVED** [review]
👍

**Pull request merged**